src/components/Person/Person.jsx

```diff
@@ -1 +1,14 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, partnerName, sex, isMarried } = person;
```

Comment:
1. [CODE STYLE] - Use destructuring for getting access to values of `props` object. Currently, you are destructuring the `person` object within the component function. Instead, you can destructure the `person` properties directly in the function parameters. This will make the code cleaner and more readable. Here is how you can do it: 
jsx
export const Person = ({
  name,
  age,
  partnerName,
  sex,
  isMarried
}) => {...}
```